README.md

```diff
@@ -117,6 +118,25 @@ $ node
 
 https://multiformats.github.io/js-multiaddr/
 
+## Resolvers
+
+`multiaddr` allows multiaddrs to be resolved when appropriate resolvers are provided. This module already has resolvers available, but you can also create your own. To provide multiaddr resolvers you can do:
```

Comment:
What do you think about clarifying the usage scope of resolvers to help people avoid dependency issues?
```suggestion
`multiaddr` allows multiaddrs to be resolved when appropriate resolvers are provided. This module already has resolvers available, but you can also create your own.  Resolvers should always be set in the same module that is calling `multiaddr.resolve()` to avoid conflicts if multiple versions of `multiaddr` are in your dependency tree. 
To provide multiaddr resolvers you can do:
```

Reply:
This is good! There was another issue regarding multiaddrs that did not have the `.resolve()` function, but for setting the resolvers this makes sense